waffle/tests/test_waffle.py

```diff
@@ -353,6 +353,22 @@ def test_testing_disabled_flag(self):
         assert not waffle.flag_is_active(request, 'foo')
         assert not hasattr(request, 'waffle_tests')
 
+    def test_testing_flag_header(self):
+        waffle.get_waffle_flag_model().objects.create(name='foo', testing=True)
+        request = RequestFactory().get('/foo', HTTP_DWFT_FOO='1')
+        request.user = AnonymousUser()
+        assert waffle.flag_is_active(request, 'foo')
+        assert 'foo' in request.waffle_tests
+        assert request.waffle_tests['foo']
+
+        # header should override cookie
```

Comment:
Update the docs to list the order of preference for test flag values. If I have a cookie, header, and querystring set, which one wins?

Reply:
@clintonb updated